templates/angular/igx-ts/custom-templates/tree-grid-finjs/files/src/app/__path__/__name__.component.ts

```diff
@@ -0,0 +1,314 @@
+import {
+    Component,
+    AfterViewInit, ElementRef, OnDestroy, OnInit, ViewChild, NgZone
+} from '@angular/core';
+import {
+    IgxButtonGroupComponent, IgxSliderComponent,
+    SortingDirection,
+    IgxTreeGridComponent,
+    PositionSettings,
+    OverlaySettings,
+    ConnectedPositioningStrategy,
+    AbsoluteScrollStrategy,
+    VerticalAlignment,
+    HorizontalAlignment
+} from 'igniteui-angular';
+import { timer } from 'rxjs';
+import { debounce } from 'rxjs/operators';
+import { LocalDataService } from './localData.service';
+import { ITreeGridAggregation } from './tree-grid-grouping.pipe';
+
+@Component({
+    providers: [LocalDataService],
+    selector: 'app-$(filePrefix)',
+	templateUrl: './$(filePrefix).component.html',
+	styleUrls: ['./$(filePrefix).component.scss']
+})
+export class GridFinTechComponent implements OnInit, AfterViewInit, OnDestroy {
+    @ViewChild('grid1', { static: true }) public grid1: IgxTreeGridComponent;
+    @ViewChild('buttonGroup1', { static: true }) public buttonGroup1: IgxButtonGroupComponent;
+    @ViewChild('slider1', { static: true }) public volumeSlider: IgxSliderComponent;
+    @ViewChild('slider2', { static: true }) public intervalSlider: IgxSliderComponent;
+
+    public theme = false;
+    public volume = 1000;
+    public frequency = 500;
+    public data: any[] = [];
+    public controls = [
+        {
+            disabled: false,
+            icon: 'update',
+            label: 'LIVE PRICES',
+            selected: false
+        },
+        {
+            disabled: false,
+            icon: 'update',
+            label: 'LIVE ALL PRICES',
+            selected: false
+        },
+        {
+            disabled: true,
+            icon: 'stop',
+            label: 'Stop',
+            selected: false
+        }
+    ];
+    public groupColumns = ['Category', 'Type', 'Contract'];
+    public aggregations: ITreeGridAggregation[] = [
+        {
+            aggregate: (parent: any, data: any[]) => {
+                return data.map((r) => r.Change).reduce((ty, u) => ty + u, 0);
+            },
+            field: 'Change'
+        },
+        {
+            aggregate: (parent: any, data: any[]) => {
+                return data.map((r) => r.Price).reduce((ty, u) => ty + u, 0);
+            },
+            field: 'Price'
+        },
+        {
+            aggregate: (parent: any, data: any[]) => {
+                return parent.Change / (parent.Price - parent.Change) * 100;
+            },
+            field: 'Change(%)'
+        }
+    ];
+    public primaryKey = 'ID';
+    public childDataKey = 'Children';
+    public groupColumnKey = 'Categories';
+
+    public items: any[] = [{ field: 'Export native' }, { field: 'Export JS Excel' }];
+
+    public _positionSettings: PositionSettings = {
```

Comment:
no need for the "_" infront of properties and it makes lint unhappy. Could you please remove them from `_timer`, `_positionSettings` & `_overlaySettings`?